internal/repository-mysql/file_repository.go

```diff
@@ -81,6 +81,14 @@ func (r *fileRepository) Save(p repository.SaveFileParam) error {
 	return err
 }
 
+func (r *fileRepository) Delete(identifier string) error {
+	_, err := r.db.Exec(
+		"DELETE FROM file WHERE unique_id = ?",
```

Comment:
Ah, I must be forgot to say that we need to implement soft-delete feature @yudium